src/main/java/BasketballGame/GameRegister.java

```diff
@@ -0,0 +1,186 @@
+                if (year == gameYear && month == gameMonth && day == gameDay && hour == gameHour) {
+                    printDate(i);
+                }
+            } else if (year != 0 && month != 0 && day != 0) {
+                if (year == gameYear && month == gameMonth && day == gameDay) {
+                    printDate(i);
+                }
+            } else if (year != 0 && month != 0) {
+                if (year == gameYear && month == gameMonth) {
+                    printDate(i);
+                }
+            }
+        }
+    }
+
+    public void printTodaysGames() {
+        for (int i = 0; i < game.size(); i++) {
+            if (localDateTime.getDayOfMonth() == (game.get(i).getGameTime().getDayOfMonth())) {
+                printDate(i);
+            }
+        }
+    }
+
+    public void printGameInformation (Game game) {
+        int [] teamOne = {PlayerNumber("One", game),PlayerNumber("Two", game), PlayerNumber("Three", game), PlayerNumber("Four", game),PlayerNumber("Five", game)};
+        int [] teamTwo = {PlayerNumberTwo("One", game),PlayerNumberTwo("Two", game), PlayerNumberTwo("Three", game),PlayerNumberTwo("Four", game),PlayerNumberTwo("Five", game)};
+        String [] teamOneNames = {PlayerName("One", game),PlayerName("Two", game),PlayerName("Three", game), PlayerName("Four", game),PlayerName("Five", game) };
+        String [] teamTwoNames = {PlayerNameTwo("One", game),PlayerNameTwo("Two", game),PlayerNameTwo("Three", game), PlayerNameTwo("Four", game),PlayerNameTwo("Five", game) };
+        System.out.println("|" + fixedLengthString(game.getTeamOne().getTeamName(),22) + fixedLengthString("|",23) + "|    | " + game.getTeamOneScore() + ":"
+                            + game.getTeamTwoScore() + " |    | " + fixedLengthString(game.getTeamTwo().getTeamName(),22) + fixedLengthString(" |",23) + "|");
+        System.out.println("|----------------------|----------------------|----|-------|----|------------------------|---------------------|");
+        System.out.println("|" + fixedLengthString(game.getTeamOne().getCoach().getCoachTitle(),22) + "|" + fixedLengthString(game.getTeamOne().getCoach().getCoach().getName(),22) + "|    |       |    |"
+                + fixedLengthString(game.getTeamTwo().getCoach().getCoachTitle(),24) + "|" + fixedLengthString(game.getTeamTwo().getCoach().getCoach().getName(),21) + "|");
+        for (int i = 0; i < teamOne.length; i++) {
+            System.out.println("|"+fixedLengthString(String.valueOf(teamOne[i]),22)+"|" + fixedLengthString(teamOneNames[i],22)+"|    |       |    |"
+                    + fixedLengthString(String.valueOf(teamTwo[i]),24)+"|"+fixedLengthString(teamTwoNames[i], 21)+"|");
+        }
+        System.out.println("|----------------------|----------------------|----|-------|----|------------------------|---------------------|");
+        System.out.println("Match started at: " + game.getGameTime().format(myFormatObj));
+
+    }
```

Comment:
Split this code into steps:
GenerateTeam(String name);
team.ToString()
PrintRow(team.ToString())
PrintTopOrBot()